server/lib/realtime/workflows/manager.ex

```diff
@@ -0,0 +1,153 @@
+defmodule Realtime.Workflows.Manager do
+  use GenServer
+  require Logger
+
+  alias Realtime.Adapters.Changes
+  alias Realtime.TransactionFilter
+  alias Realtime.Workflows
+
+  defmodule State do
+    defstruct [:state, :queue, :table]
+  end
+
+  @table_name :workflows_manager
+
+  ## Manager API
+
+  def start_link(config) do
+    GenServer.start_link(__MODULE__, config, name: __MODULE__)
+  end
+
+  @doc """
+  Send notification for txn to the workflow manager.
+  """
+  def notify(txn) do
+    GenServer.cast(__MODULE__, {:notify, txn})
```

Comment:
@fracek we should consider changing this to a `call`:

```suggestion
    GenServer.call(__MODULE__, {:notify, txn})
```

I'm worried about `GenServer.cast` (`Realtime.Workflows.Manager.notify(txn)`) async message because `Realtime.Workflows.Trigger.notify(txn)` is called directly afterwards and it depends on `Manager` to keep `:ets` table up to date.